content/javascript/concepts/loops/loops.md

```diff
@@ -159,6 +159,24 @@ for (let i = 0; i < array.length; i++) {
 // Output: Every item in the array
 ```
 
+## Looping Through Objects
+
+To loop through objects use `Object.entries` which will first convert the objects into an array and then loop through that array. 
+
+Using `Object.entries` an array of arrays is created which will contain both the property and the value respectively.
+
+```js
+const objectK = {
+  name: 'Codecademy',
+  age: 10
+};
```

Comment:
It seems we're missing the loop code that uses `Object.entries()`.

Reply:
```js
for(const [key, value] of Object.entries(objectK)) {
  console.log(`${key}: ${value}`);
}
```

Reply:
@KabirCampwala, please refer to @Dusch4593's comment above as to what the js code block is missing!

I'd suggest to add `myEntries = Object.entries(objectK))` and `console.log(myEntries)` or just `console.log(Object.entries(objectK))` first with the output `[ [ 'name', 'Codecademy' ], [ 'age', 10 ] ]` commented out, then do the `for` loop suggested above:

```js
for(const [key, value] of Object.entries(objectK)) {
  console.log(`${key}: ${value}`);
};
```
Finally, we would add the output commented out on the following lines. This way, we won't need a separate block for the outputs.

In addition, to make these clear to the reader, we should add a line about what is happening before/after the code block. You can determine where you wish to place this explanation.